content/documentation/pages/4-batch-developer-guides/2-batch/9-data-flow-simple-task-cloudfoundry.md

```diff
@@ -0,0 +1,259 @@
+
+For example, up until now we have run the 2 example tasks individually. Howeverm using Spring Cloud Data Flow and the [Composed Task](http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-composed-tasks) feature, we can execute both Task in one flow.
+
+When using Spring Cloud Dataflow to deploy task and stream applications to Cloud Foundry, you have 2 options on how to run Spring Cloud Data Flow itself:
+
+- **locally** and target a remote Cloud Foundry instance or
+- **on Cloud Foundry** itself and deloy applications there as well
+
+For this guide, we will deploy Spring Cloud Data Flow to Cloud Foundry.
+
+**IMPORTANT** This guide will provide you with quick steps to get you going. For more specific information, please refer to the [Cloud Foundry specific section](https://docs.spring.io/spring-cloud-dataflow/docs/current-SNAPSHOT/reference/htmlsingle/#getting-started-cloudfoundry) of the Spring Cloud Data Flow reference documentation in regards to how to set things up for Cloud Foundry.
+
+# Requirements
+
+On your local machine, you will need to have installed:
+
+- Java
+- [Git](https://git-scm.com/)
+
+Please also make sure that you have the [Cloud Foundry command line interface](https://console.run.pivotal.io/tools) installed ([documentation](https://docs.run.pivotal.io/cf-cli/)).
+
+## Local Project Setup
+
+The example consists of 2 applications:
+
+- billsetuptask
+- billrun
+
+The first task application `billsetuptask` will create an SQL table that is needed by the second task application.
+
+```sql
+CREATE TABLE IF NOT EXISTS BILL_STATEMENTS (
+    id int,
+    first_name varchar(50),
+    last_name varchar(50),
+    minutes int,
+    data_usage int,
+    bill_amount decimal(10,2)
+);
+```
+
+The second task ([Spring Batch](https://spring.io/projects/spring-batch)) application `billrun` will read JSON files containing simple user cellphone usage data from a specified directory, price it and place the results in the previously created database table.
+
+They are both located in the same Git repository. In order to get started, please checkout the project using:
+
+```bash
+$ git clone https://github.com/spring-cloud/spring-cloud-dataflow-samples.git
+$ cd spring-cloud-dataflow-samples/dataflow-website/batch-developer-guides/batch/batchsamples
+```
+
+Let's first build the entire project:
+
+```bash
+$ ./mvnw clean package
+```
+
+This will create 2 executable Jar files, one for each application:
+
+- billsetuptask/target/**billsetuptask-0.0.1-SNAPSHOT.jar**
+- billrun/target/**billrun-0.0.1-SNAPSHOT.jar**
+
+In order for Spring Cloud Data Flow to consume the applications, they need to be accessibly via HTTP(S). An easy solution is to add the 2 jar files to a GitHub repository.
+
+We can then reference the 2 applications using Urls like the following:
+
+- https://github.com/ghillert/task-apps/raw/master/billsetuptask-0.0.1-SNAPSHOT.jar
```

Comment:
You may have meant this.   But these links return 404's .  I got this to work https://github.com/ghillert/task-apps/blob/master/billsetuptask-1.0.0.BUILD-SNAPSHOT.jar?raw=true.  May see if we want to host these on the sample site.

Reply:
Should be fixed